content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-iptables.adoc

```diff
@@ -207,3 +207,28 @@ do SSL or not)
 firewalld will then create the required iptables rules so that incoming
 connections on port 80 are forwarded to jenkins on 8080 (and 443 is
 forwarded to 8443).
+
+
+== Unix domain sockets support
+
+The addition of Unix domain sockets support for Jenkins when configured behind an Iptables reverse proxy. 
+This feature allows for enhanced communication between Jenkins and the reverse proxy, 
+improving performance and security.
+
+=== What does Unix domain sockets support mean for Iptables?
+
+Support for Unix domain sockets in iptables allows for the filtering of traffic based on Unix domain sockets. However, it's crucial to recognize that this capability is typically deactivated by default in iptables. Unix domain sockets are predominantly utilized for local inter-process communication and binding, serving as endpoints for communication between processes within the same operating system. If connectivity issues arise with software relying on Unix domain sockets, the root cause likely extends beyond iptables configurations. In such instances, detailed error messages and specific information are pivotal for effective troubleshooting. When integrated with iptables, Unix domain sockets are treated similarly to other socket types, enabling administrators to define firewall rules regulating communication over these sockets.
+
+=== Example Configuration:
+To configure Unix domain sockets with Iptables for Jenkins, you can use the following example code:
+
+[source]
+----
+# Redirect traffic from port 80 to the Unix domain socket
+iptables -t nat -A PREROUTING -i eth0 -p tcp --dport 80 -j REDIRECT --to-ports <socket_path>
+# Redirect traffic from port 443 to the Unix domain socket
+iptables -t nat -A PREROUTING -i eth0 -p tcp --dport 443 -j REDIRECT --to-ports <socket_path>
```

Comment:
The commands don't work as listed.  The `--to-ports` command line argument expects a port or a port range.  It won't accept a path to a Unix domain socket.  As far as I can tell from the iptables documentation on my Debian Linux, there is no way to have iptables directly configure to use a Unix domain socket.

Reply:
Darin Pope has provided a video that configures a Unix domain socket on a Red Hat Enterprise Linux derivative with Apache as the reverse proxy. 

https://www.youtube.com/watch?v=KxFJh4184vk